.github/workflows/update-webassets.yaml

```diff
@@ -28,6 +28,7 @@ jobs:
           repository: gravitational/webassets-fork # TODO(zmb3): switch off the fork
           submodules: recursive
           path: dist
+          ref: ${{ github.ref_name }}
```

Comment:
Might be worth adding a comment here because I don't think this will work on `webapps` because we use different branch names there.

Reply:
Nope, we use the same branch names.